setup.py

```diff
@@ -5,32 +5,39 @@
     from distutils.core import setup
 
 config = {
-    'description': "Parse Ruby's Gemfiles",
-    'author': 'Balasankar C',
-    'url': 'https://gitlab.com/balasankarc/gemfileparser',
-    'download_url': 'https://gitlab.com/balasankarc/gemfileparser',
-    'author_email': 'balasankarc@autistici.org',
+    'description': "A library to parse Rubygem gemspec and Gemfile files and Cocoapods podspec and Podfile files using Python. Friendly fork of https://gitlab.com/balasankarc/gemfileparser",
+    'author': 'nexB',
```

Comment:
Please keep the original author name there. Or something like `Rohit Potter for nexB based on original work of Balasankar C' ... but in all cases we want Balasankar to get proper credits

Reply:
Okh :)
I will change this in new commits.

Reply:
In case we get to collaborate and move development upstream as suggested in https://github.com/nexB/gemfileparser2/pull/2#issuecomment-663682411, this can definitely change to an AUTHOR file based logic so that every contributor gets credit. :slightly_smiling_face: